test/ci.sh

```diff
@@ -31,6 +31,7 @@ version_checks() {
   version=$(get_version)
   grep "version.*$version" setup.py
   grep "version.*$version" rpm/python-ramalama.spec
+  grep "version.*$version" ramalama/version.py
```

Comment:
I hope this part of the CI is actually running, parts of our CI tend to disappear at times.

Reply:
We need to make this easier, currently this constant is defined in 3 other places.

pyproject.toml, setup.py, rpm/python-ramalama.spec

Need a release script top populate everything.